.pairs

```diff
@@ -1,17 +1,27 @@
 # .pairs - configuration for 'git pair'
 pairs:
   # <initials>: <Firstname> <Lastname>[; <email-id>]
-  jh: Jenny Heath; jheath
-  bg: Ben Golder; bgolder
-  td: Tom Dooner; tdooner
-  lk: Laura Kogler; lkogler
-  mtm: Molly T-M; mollyt
-  jg: Jonathan Greenberg; jgreenberg
-  bv: Ben Vandgrift; bvandgrift
-  dbd: Dimitri DeFigueiredo; ddefigueiredo
-  al: Asheesh Laroia; alaroia
-email:
-  prefix: pair
-  domain: codeforamerica.org
-  no_solo_prefix: true
```

Comment:
hm actually sorry for the extra comment but i'm not sure whether we still need prefix/no_solo_prefix ?

Reply:
It's interesting because in this new format, it choses one author at random for the email address. (because there are different domains now, rather than putting "pair+david+sam@codeforamerica.org" it will simply chose "david@codeforamerica.org" or "sam@codeforamerica.org"). It will, however, put both members as the commit author. So we shouldn't need the no_solo_prefix anymore.

more details.. https://github.com/pivotal-legacy/git_scripts#git-pair-commit